api/v6/products.thrift

```diff
@@ -35,7 +35,7 @@ struct Product {
   2: string            endpoint,
   3: string            displayedName_b64,
   4: string            description_b64,
-  5: bool              connected,         // !DEPRECATED FLAG it will be always False.
+  5: bool              connected,         // True only if database status is OK.
```

Comment:
Just because the value is set, doesn't mean that the variable's usage itself isn't deprecated.